OneSignalSDK/onesignal/in-app-messages/src/main/java/com/onesignal/inAppMessages/internal/InAppMessagesManager.kt

```diff
@@ -350,6 +350,7 @@ internal class InAppMessagesManager(
         // If there was a message dequeued, display it
         if (messageToDisplay != null) {
             var result = _displayer.displayMessage(messageToDisplay!!)
+            _state.inAppMessageIdShowing = null
```

Comment:
If result isn't null or false that means that we are displaying a message correct? In that case setting the id to null seems wrong.